pkg/views/workspace/info/view.go

```diff
@@ -169,12 +169,21 @@ func getInfoLineGitStatus(key string, status *apiclient.GitStatus) string {
 	}
 
 	unpushedOutput := ""
-	if *status.Ahead > 0 && *status.Behind > 0 {
-		unpushedOutput = fmt.Sprintf(" (%d commits ahead, %d commits behind)", *status.Ahead, *status.Behind)
-	} else if *status.Ahead > 0 {
-		unpushedOutput = fmt.Sprintf(" (%d commits ahead)", *status.Ahead)
-	} else if *status.Behind > 0 {
-		unpushedOutput = fmt.Sprintf(" (%d commits behind)", *status.Behind)
+
+	if *status.Ahead > 0 {
```

Comment:
We should add a nil check here as well just in case.

(Same thing for Behind)

Reply:
done!

Reply:
@bryans-go the only change that was required was adding `status.Ahead != nil &&` 😅 

Everything else from the latest commit can be reverted.